public/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
    <script src="https://as.alipayobjects.com/g/component/??es6-shim/0.35.1/es6-sham.min.js,es6-shim/0.35.1/es6-shim.min.js"></script>
-
+   <script src="lodash.js"></script>
```

Comment:
do this by importing

Reply:
and delete the lodash.js file which u imported

Reply:
ok sir